TeamCode/src/main/java/org/firstinspires/ftc/teamcode/helpers/commands/wrappers/setArmAngle.java

```diff
@@ -0,0 +1,16 @@
+package org.firstinspires.ftc.teamcode.helpers.commands.wrappers;
+
+import com.arcrobotics.ftclib.command.InstantCommand;
+import org.firstinspires.ftc.teamcode.subsystems.arm.ArmConfiguration;
+import org.firstinspires.ftc.teamcode.subsystems.arm.ArmSubsystem;
+
+public class setArmAngle extends InstantCommand {
+
+    public setArmAngle(ArmSubsystem arm, ArmConfiguration.TargetAngle angle) {
```

Comment:
```suggestion
    public SetArmAngle(ArmSubsystem arm, ArmConfiguration.TargetAngle angle) {
```

Same here for the subsystem being passed in